src/driver/src/DefaultDriver.cpp

```diff
@@ -89,7 +201,26 @@ genny::driver::DefaultDriver::OutcomeCode doRunLogic(
     auto actorSetup = metrics.operation("Genny", "Setup", 0u);
     auto setupCtx = actorSetup.start();
 
-    auto yaml = loadConfig(options.workloadSource, options.workloadSourceType);
+    auto config = loadConfig(options.workloadSource, options.workloadSourceType);
+    fs::path phaseConfigSource;
+    if (options.workloadSourceType == DefaultDriver::ProgramOptions::YamlSource::kString) {
+        phaseConfigSource = fs::current_path();
+    } else {
+        /*
+         * The directory structure for workloads and phase configs is as follows:
+         * /etc (or /src if building without installing)
+         *     /workloads
+         *         /[name-of-workload-theme]
+         *             /[my-workload.yml]
+         * The path to the phase config snippet is obtained as a relative path from the workload
+         * file.
+         */
+        phaseConfigSource = fs::path(options.workloadSource).parent_path();
+    }
+
+    std::cout << YAML::Dump(yaml) << std::endl;
+    return genny::driver::DefaultDriver::OutcomeCode::kSuccess;
```

Comment:
this code modifies `phaseConfigSource` but then the code that actually does `YAML::Dump` doesn't use that - is there some magic I'm missing?

Reply:
I left in some debugging code :(